Configure

```diff
@@ -464,6 +465,7 @@ our %disabled = ( # "what"         => "comment"
                   "ssl-trace"           => "default",
                   "ssl3"                => "default",
                   "ssl3-method"         => "default",
+                  "tfo"                 => "default",
```

Comment:
I am sceptical about having this as a config time option. But if we do have it then it should be added to INSTALL.